.github/workflows/vrt.yml

```
@@ -23,12 +26,12 @@ permissions:
 
 jobs:
     vrt:
+        if: ${{ github.event.push.ref != 'refs/heads/changeset-release' ||  github.event.push.ref != 'refs/heads/dependabot'}}
```

```suggestion
        if: ${{ !contains(github.event.push.ref, 'changeset-release') &&  !contains(github.event.push.ref, 'dependabot') }}
```

Should this be a not contains?

---

oh perhaps!